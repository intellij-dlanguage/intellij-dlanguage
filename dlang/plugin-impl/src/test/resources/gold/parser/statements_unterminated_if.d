void main() {

    if (/*Comment started
}