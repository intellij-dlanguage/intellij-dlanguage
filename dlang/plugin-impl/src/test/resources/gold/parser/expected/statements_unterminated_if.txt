D Language File
  DLanguageDeclarationImpl(DECLARATION)
    FUNCTION_DECLARATION
      DLanguageTypeImpl(TYPE)
        DLanguageType_2Impl(TYPE_2)
          DLanguageBuiltinTypeImpl(BUILTIN_TYPE)
            PsiElement(DlangTokenType.void)('void')
      IDENTIFIER
        PsiElement(DlangTokenType.ID)('main')
      DLanguageParametersImpl(PARAMETERS)
        PsiElement(DlangTokenType.()('(')
        PsiElement(DlangTokenType.))(')')
      DLanguageFunctionBodyImpl(FUNCTION_BODY)
        DLanguageSpecifiedFunctionBodyImpl(SPECIFIED_FUNCTION_BODY)
          DLanguageBlockStatementImpl(BLOCK_STATEMENT)
            PsiElement(DlangTokenType.{)('{')
            DLanguageDeclarationsAndStatementsImpl(DECLARATIONS_AND_STATEMENTS)
              DLanguageDeclarationOrStatementImpl(DECLARATION_OR_STATEMENT)
                DLanguageStatementImpl(STATEMENT)
                  DLanguageStatementNoCaseNoDefaultImpl(STATEMENT_NO_CASE_NO_DEFAULT)
                    DLanguageIfStatementImpl(IF_STATEMENT)
                      PsiElement(DlangTokenType.if)('if')
                      PsiElement(DlangTokenType.()('(')
            PsiErrorElement:Expected } instead of EOF
              <empty list>
    PsiComment(DlangTokenType.BLOCK_COMMENT)('/*Comment started\n}')